services/core-web/src/components/Forms/reports/ReportPermitRequirementForm.tsx

```diff
@@ -36,22 +36,23 @@ interface ReportPermitRequirementProps {
   mineReportPermitRequirement?: IMineReportPermitRequirement;
   canEditPermitConditions: boolean;
   refreshData: () => Promise<void>;
+  currentAmendment: IPermitAmendment;
   mineGuid: string;
 }
 
 export const ReportPermitRequirementForm: FC<ReportPermitRequirementProps> = ({
   onSubmit,
   condition,
-  permitGuid,
   modalView = true,
   mineReportPermitRequirement,
   canEditPermitConditions,
   refreshData,
+  permitGuid,
+  currentAmendment,
   mineGuid,
 }) => {
   const dispatch = useDispatch();
   const [isEditMode, setIsEditMode] = React.useState(modalView);
-  const latestPermitAmendment = useSelector(getLatestAmendmentByPermitGuid(permitGuid));
```

Comment:
A lot of these changes are just to make it so that we can view conditions for the latest completed amendment, AND the latest one that hasn't the been reviewed